packages/webpack-cli/lib/utils/cli-flags.js

```diff
@@ -34,8 +34,11 @@ let flagsFromCore = version.startsWith(5)
 const duplicateFlags = ['entry', 'stats', 'watch', 'target'];
```

Comment:
We should remove duplicate flags from our flags in favor core flags, but we can postpone it and do it in the other PR

Reply:
I'm afraid if we can do this we are still supporting webpack 4, and if we remove these flags from our flags we will lose compatibility with v4, and users using v4 will no longer be able to access these flags.

Reply:
@snitin315 We can do it only for webpack@5

Reply:
oh yes, will implement in separate PR :+1: